DeepFilterNet/df/scripts/export.py

```diff
@@ -303,12 +304,17 @@ def main(args):
     )
     sample = get_test_sample(df_state.sr())
     enhanced = enhance(model, df_state, sample, True)
-    save_audio("out/enhanced.wav", enhanced, df_state.sr())
-    if not os.path.isdir(args.export_dir):
-        os.makedirs(args.export_dir)
+    try:
+        # attempt saving enhanced audio
+        save_audio("out/enhanced.wav", enhanced, df_state.sr())
+    except RuntimeError:
```

Comment:
Can we instead wrap this into a condition on `out_dir.is_dir():`

Reply:
Yes, I'll do this